src/Conversion/ONNXToTOSA/Flow/EntryPoint.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===---------------- EntryPoint.cpp - EntryPoint Op --------------------===//
+//
+// Copyright (c) 2022 Advanced Micro Devices, Inc.
+//
+// =============================================================================
+//
+// This file lowers ONNX EnryPoint to TOSA dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/Tosa/IR/TosaOps.h"
+#include "src/Conversion/ONNXToTOSA/ONNXToTOSACommon.hpp"
+#include "src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.hpp"
+#include "src/Dialect/ONNX/ONNXOps.hpp"
+
+using namespace mlir;
+
+namespace onnx_mlir {
+
+namespace {
+
+class ONNXEntryPointLoweringToTOSA
+    : public OpConversionPattern<ONNXEntryPointOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  using OpAdaptor = typename ONNXEntryPointOp::Adaptor;
+  // This function if from the typesTransformsToTorchPass.cpp
```

Comment:
I'm not sure I understand your comment

Reply:
yeah there is a typo. 'if' should be 'is'